lib/movable_ink/aws/ssm.rb

```diff
@@ -1,12 +1,26 @@
 module MovableInk
   class AWS
     module SSM
-      def ssm
-        @ssm_client ||= Aws::SSM::Client.new(region: 'us-east-1')
+      def us_east_ssm
```

Comment:
In other modules where we have clients for multiple regions, we've generally passed in the region as a parameter.  See https://github.com/movableink/awslib/blob/master/lib/movable_ink/aws/autoscaling.rb#L4 for example.  Is there a reason you're preferring to create explicit methods per region?

Reply:
I assume the original implementation was written that way because us-east-1 is the only region with parameters, and I kept explicit regions to treat `us-east-1` as a source of truth and fall back to replicas in `us-west-2`. 

Would you prefer we setup parameter replication to all the regions where we have instances, allow the region to be passed in and use any other regions as backups?